data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -82,7 +82,12 @@
     "Life expectancy (years) (percentile)",
     "Traffic proximity and volume (percentile)",
     "FEMA Risk Index Expected Annual Loss Score (percentile)",
-    "Energy burden (percentile)",
+    # this column is not coherent and is rather vague and should be changed to the original
+    # The question is where do we change this? I have thoughts on this
+    # but am seeking definitive feedback that is aligned with how this
+    # codebase was originally created
+    "BURDEN", 
```

Comment:
@ahlusar1989 -- you should just rename this in the ETL for DOE LEAD. I should've noticed this on the original PR, but you'll need to update these lines: 

```

        output_df = self.raw_df.rename(
            columns={
                "AvgEnergyBurden": self.ENERGY_BURDEN_FIELD_NAME,
                self.TRACT_INPUT_COLUMN_NAME: self.GEOID_TRACT_FIELD_NAME,
            }
        )
```

Reply:
BTW I just created this for the backlog, after Tuesday we can do some cleanup: https://github.com/usds/justice40-tool/issues/977.

Reply:
Cool - thanks for confirming @lucasmbrown-usds. Just wanted to make sure this was what needed to update here.

Reply:
@ahlusar1989 -- why did you mark this as resolved? For code clarity we still need to rename `"BURDEN"` to self.ENERGY_BURDEN_FIELD_NAME.